python/torch_mlir/extras/fx_importer.py

```diff
@@ -315,15 +315,23 @@ def is_builtin_function_or_method(obj: Any) -> bool:
     return isinstance(obj, (BuiltinMethodType, BuiltinFunctionType))
 
 
-@dataclass(frozen=True, slots=True)
+@dataclass(frozen=True)
 class InputInfo:
     """Provides additional metadata when resolving inputs."""
 
+    __slots__ = [
```

Comment:
nit: perhaps add a `TODO` here to switch back to using `slots=True` when py-3.9 support is dropped?

Reply:
done